homeassistant/components/tesla/sensor.py

```diff
@@ -27,7 +27,7 @@
         if device.bin_type == 0x4:
             entities.append(TeslaSensor(device, controller, config_entry, "inside"))
             entities.append(TeslaSensor(device, controller, config_entry, "outside"))
-        elif device.bin_type in [0xA, 0xB, 0x5]:
+        else:
```

Comment:
Will this change start including new sensor types besides the one you desired to add originally?

Reply:
No. Originally it was an else statement but I had to add this `elif` to avoid adding in the charging rate sensor in the configflow/async PR. Martin had me break it down into multiple PRs.